internal/testing/enginetest/enginetest.go

```diff
@@ -212,36 +212,232 @@ func RunTestModuleEngine_Call_HostFn(t *testing.T, et EngineTester) {
 		require.Equal(t, uint64(3), results[0])
 		require.Same(t, memory, ctxMemory)
 	})
+}
 
-	t.Run("errs when not enough parameters", func(t *testing.T) {
-		_, err := me.Call(modCtx, f)
-		require.EqualError(t, err, "expected 1 params, but passed 0")
-	})
+func RunTestModuleEngine_Call_HostFn(t *testing.T, et EngineTester) {
+	runTestModuleEngine_Call_HostFn_ModuleContext(t, et) // TODO: refactor to use the same test interface.
 
-	t.Run("errs when too many parameters", func(t *testing.T) {
-		_, err := me.Call(modCtx, f, 1, 2)
-		require.EqualError(t, err, "expected 1 params, but passed 2")
-	})
+	e := et.NewEngine()
+
+	imported, importedMe := setupCallTests(t, e)
```

Comment:
FYI the "importing" tests from the other PR are the issue. only one module is ok (well as long as tests pass in CI :D)